client/lib/js/react/components/accounts.js

```diff
@@ -74,51 +72,40 @@ const AccountList = React.createClass({
     }
 });
 
-const AccountRow = React.createClass({
+const AccountWrapper = React.createClass({
     propTypes: {
         account: React.PropTypes.object,
         activeAccount: React.PropTypes.object,
+        children: React.PropTypes.oneOfType([
+            React.PropTypes.arrayOf(React.PropTypes.node),
+            React.PropTypes.node
+        ]),
         handleClick: React.PropTypes.func
     },
-    
-    connectToLedger() {
-        const { account } = this.props;
-        return new BigchainDBLedgerPlugin({
-            auth: {
-                account: {
-                    id: account.vk,
-                    uri: 'ws://localhost:8888/users/' + account.vk
-                }
-            },
-        });
-    },
-    
+
     handleClick() {
         const { account, handleClick } = this.props;
-        const ledger = this.connectToLedger(account);
-        ledger.connect().catch((err) => {
-            console.error((err && err.stack) ? err.stack : err);
-        });
-
-        handleClick(account, ledger);
+        handleClick(account);
```

Comment:
Either make the propType for handleClick be isRequired or wrap this around a safeInvoke.